web/ui/module/codemirror-promql/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+## Prometheus Community Code of Conduct
```

Comment:
We don't need this file as we have it in the Prometheus repo root already.

Reply:
same reason as for the ci configuration. It's here for the mirror

Reply:
Ah yes, that makes sense. Keep these then.

Reply:
Btw. these files are normally synchronized between all repos: https://github.com/prometheus/prometheus/blob/e2557e4920b46792c640dea59b96cc39e9db1964/scripts/sync_repo_files.sh#L40

We should be sure to synchronize them into this subdir as well, otherwise they will fall out of date at some point.